packages/atomic/src/components/atomic-breadcrumb-manager/atomic-breadcrumb-manager.tsx

```diff
@@ -160,13 +166,31 @@ export class AtomicBreadcrumbManager implements InitializableComponent {
   private get numericFacetBreadcrumbs() {
     return this.breadcrumbManagerState.numericFacetBreadcrumbs.map(
       (breadcrumb) =>
-        this.getBreadcrumbValues(this.formatRangeBreadcrumb(breadcrumb))
+        this.getBreadcrumbValues(
+          this.formatRangeBreadcrumb(breadcrumb, (value) => {
+            const {language} = this.bindings.i18n;
+            return `${value.start.toLocaleString(
+              language
+            )} - ${value.end.toLocaleString(language)}`;
+          })
+        )
     );
   }
 
   private get dateFacetBreadcrumbs() {
-    return this.breadcrumbManagerState.dateFacetBreadcrumbs.map((breadcrumb) =>
-      this.getBreadcrumbValues(this.formatRangeBreadcrumb(breadcrumb))
+    return this.breadcrumbManagerState.dateFacetBreadcrumbs.map(
+      (breadcrumb) => {
+        const dateFormat = this.getFormat(breadcrumb.facetId);
+        return this.getBreadcrumbValues(
+          this.formatRangeBreadcrumb(
+            breadcrumb,
+            (value) =>
+              `${dayjs(value.start).format(dateFormat)} - ${dayjs(
```

Comment:
Same comment here relating to the dash. Perhaps we could format the dates, and then interpolate those values in a localized string that can be configured? Not sure ...